src/components/controllers/tutorial/index.jsx

```diff
@@ -8,12 +8,11 @@ import {
 	useMemo,
 	useCallback
 } from 'preact/hooks';
-import { useLocation, useRoute } from 'preact-iso';
+import { useLocation } from 'preact-iso';
 import { TutorialContext, SolutionContext } from './contexts';
 import { ErrorOverlay } from '../repl/error-overlay';
 import { parseStackTrace } from '../repl/errors';
 import cx from '../../../lib/cx';
-import { InjectPrerenderData } from '../../../lib/prerender-data';
```

Comment:
This was a relic of the Preact-CLI setup, `InjectPrerenderData` was linked to a `getPrerenderData` function that hasn't been in use since we switched over. Injecting therefore did nothing.